cmd/orb_test.go

```diff
@@ -611,6 +611,85 @@ var _ = Describe("Orb integration tests", func() {
 				Eventually(session).Should(gexec.Exit(0))
 				Expect(testServer.ReceivedRequests()).Should(HaveLen(2))
 			})
+
+		})
+
+		Describe("when listing orbs with a namespace argument", func() {
+			BeforeEach(func() {
+				command = exec.Command(pathCLI,
+					"orb", "list", "circleci",
+					"--host", testServer.URL(),
+					"--verbose",
+				)
+			})
+
+			It("makes a namespace query and requests all orbs on that namespace", func() {
+				By("setting up a mock server")
+
+				// These requests and responses are generated from production data,
+				// but using a 5-per-page limit instead of the 20 requested.
+				tmpBytes, err := ioutil.ReadFile(filepath.Join("testdata/gql_orb_list_with_namespace", "first_request.json"))
+				Expect(err).ShouldNot(HaveOccurred())
+				firstGqlRequest := string(tmpBytes)
```

Comment:
This test would probably be more readable with some additional test helpers.

Ex: using https://godoc.org/gotest.tools/golden#Get

```go
firstGqlRequest := golden.Get(GinkgoT(), filepath.FromSlash("gql_orb_list_with_namespace/first_request.json"))
```

`golden.Get()` handles the errors.

Reply:
`FromSlash` for Windows compatibility, very nice 👍